doc/components/popup.md

```diff
@@ -0,0 +1,481 @@
+  - Position is calculated from the top-left corner
+  - For `percentage`, position is calculated according to the option `relative`
+      - If `relative` is set to `"buf"` or `"cursor"`, `percentage string` is not allowed
+
+#### Usage
+
+```lua
+position = 50,
+```
+
+```lua
+position = "50%",
+```
+
+```lua
+position = {
+    row = 30,
+    col = 20
+}
+```
+
+```lua
+position = {
+    row = "30%",
+    col = "20%"
+}
+```
+
+---
+
+### `size`
+
+- **Type:** `number` or `percentage string` or `table`
+- **Description:**
+  - Determines the size of the popup
+  - For `percentage`, `size` is calculated according to the option `relative`
+    - If `relative` is set to `buf` or `cursor`, window `size` is considered
+  - If `size` is `number` or `percentage`, it applies to both `width` and `height`
+        - You can also pass a table to set them separately.
+
+#### Usage
+
+```lua
+size = 50,
+```
+
+```lua
+size = "50%",
+```
+
+```lua
+size = {
+    width = 30,
+    height = 50,
+}
+```
+
+```lua
+size = {
+    width = "30%",
+    height = "50%",
+}
+```
+
+---
+
+### `enter`
+
+- **Type:** `boolean`
+- **Description:** If `true`, the popup is auto-entered after mount
+
+#### Usage
+
+```lua
+enter = true,
+```
+
+---
+
+### `focusable`
+
+- **Type:** `boolean`
+- **Description:** Is `false`, the popup can not be entered by user actions (wincmds, mouse events)
+
+#### Usage
+
+```lua
+focusable = true,
+```
+
+---
+
+### `zindex`
+
+- **Type:** `number`
+- **Description:**
+  - Sets the order of the popup on z-axis
+  - Popup with higher the `zindex` goes on top of popups with lower `zindex`
+
+#### Usage
+
+```lua
+zindex = 20,
+```
+
+---
+
+### `buf_options`
+
+- **Type:** `table`
+- **Description:** Contains all buffer related options
+
+#### Usage
+
+```lua
+buf_options = {
+    modifiable = false,
+    readonly = true,
+},
+```
+
+#### Descriptions
+
+| Value        | Description                                                                  |
+| ------------ | ---------------------------------------------------------------------------- |
+| `modifiable` | If `false` the buffer contents cannot be changed. <br> Check `:h modifiable` |
+| `readonly`   | If `true`, writes fail unless you use a `!` <br> Check `:h readonly`         |
+
+---
+
+### `win_options`
+
+- **Type:** `table`
+- **Description:** Contains all window related options
+
+#### Usage
+
+```lua
+win_options = {
+    winblend = 10,
+    winhighlight = "Normal:Normal",
+},
+```
+
+#### Descriptions
+
+| Value          | Description                                                                               |
+| -------------- | ----------------------------------------------------------------------------------------- |
+| `winblend`     | Transparency for a floating window. <br> Goes from `0` to `100`. <br> Check `:h winblend` |
+| `winhighlight` | Window local highlights. <br> Check `:h winhighlight`                                     |
```

Comment:
Describing these here seems unnecessary, since `win_options` can take other window optinos too (not just these two).

Reply:
Same answer as above, would be nice to point the user to a list of all of them if possible.

Reply:
Do `:help options` and search for `local to window`